custom-words.txt

```diff
@@ -1749,3 +1749,9 @@ cloudManifestFiles
 highestcriticalimpact
 apiserver
 kubeadmin
+userprincipalname
+sessionstate
+registation
```

Comment:
don't need this - should be registration

Reply:
ok